test/y2storage/boot_requirements_errors_test.rb

```diff
@@ -257,6 +257,21 @@
 
     let(:zipl_partition) { partition_double("/dev/sda1") }
 
+    let(:efiboot) { true }
+
+    it "contains an error when there is /boot that is not big enough" do
```

Comment:
IMHO there is too much mocking here. As probed by the fact that you were using the wrong method to find the mount point and it was still passing.

Reply:
@ancorgs it is in whole this test. To be honest I prefer here if real graph from xml or yaml is used instead, but currently it will be quite big change.

Reply:
Ok, let's leave it as it is. But please do some manual test with the `partitioner_testing` client